installation/resources/kyma/installer-overrides-tracing.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tracing-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: tracing
+    kyma-project.io/installation: ""
+data:
+  global.tracing.enabled: "true"
```

Comment:
This are the same values as the defaults from kyma we can remove the whole file.

Reply:
There is a kyma overrides that change this value to "false" so we need to override it again to "true"